software/cloud-dashboard/react-client/src/components/temp-view.js

```diff
@@ -17,13 +17,20 @@ class TempView extends Component {
 	};
 
 	render() {
+    var tempDataMin = 0;
+    var tempDataMax = 80;
+
 		var tempData = [];
 		if (this.props.temps.data) {
             tempData = this.props.temps.data.map(currentItem => {
                 return {...currentItem.temps, time: currentItem.time};
             });
             tempData = tempData.reverse()
         }
+
+    this.props.dataMin = (tempData.min() < tempDataMin ? tempDataMin : tempData.min());
```

Comment:
@suyashkumar trying to set these here seems to break the plots... insights?

Reply:
nvm - I realize this is broken b/c of trying to assign something to `props` incorrectly (I think)

Reply:
ah yes, you should not be setting this components own props from within it -- the parent to this component passes in props to this component somewhat like function inputs. 

Likely what you want to be doing is passing in `dataMin` and `dataMax` as input props to the plot itself right? In which case you would be passing them into `PlotCard` like `data` is being passed in on L56.

Reply:
yep... I realized this was a broken approach.  not surprised by that.